packages/react/src/components/Input/index.js

```diff
@@ -25,6 +28,14 @@ const Input = React.forwardRef(
       'is-invalid': isInvalid,
     };
 
+    const getInputIcon = () => {
```

Comment:
You don't have to prefix everything, you already are in an Input context 😉 

```suggestion
    const getIcon = () => {
```